src/redux/reducers/experiments/experimentsMetadataRename.js

```diff
@@ -0,0 +1,18 @@
+const experimentsMetadataRename = (state, action) => {
+  const { oldKey, newKey, experimentId } = action.payload;
+
+  const { metadataKeys } = state[experimentId];
+  const newMetadataKeys = metadataKeys.map((metadataKey) => (
+    metadataKey === oldKey ? newKey : metadataKey
+  ));
+
+  return {
+    ...state,
+    [experimentId]: {
+      ...state[experimentId],
+      metadataKeys: newMetadataKeys,
+    },
+  };
+};
+
+export default experimentsMetadataRename;
```

Comment:
This is just a rename of `experimentsMetadataUpdate`, one extra thing I did here is deal with the annoying thing where if we renamed a metadata track key, it would push the renamed key to the end.

Look at src/redux/reducers/experiments/experimentsMetadataUpdate.js to see what was changed